package.json

```diff
@@ -1,10 +1,14 @@
 {
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
Actually, the third version is not released yet, let's postpone version changing

Reply:
fixed